news/2019-09-07-osu!taiko-mapping-olympiad-2-results.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #2 Results"
+date: 2019-09-07 17:00:00 +0000
+---
+
+The judging for the first osu!taiko Mapping Olympiad contest has been completed and the results are in: congratulations to the first **osu!taiko Mapping Olympian** and taiko's newest *Elite Mapper*!
+
+[![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)](https://osu.ppy.sh/community/contests/65)
+
+For the inaugural taiko Olympiad contest, we tasked mappers to create a full spread showcasing their proficiency in taiko mapping on the song *Bwa !* from [**Thaehan**](https://osu.ppy.sh/beatmaps/artists/7). There were a total of 33 entries and below are the results for the **top 10 entries**:
+
+| Rank 	| Username  	| Entry Name          	| Flow and Structure (30) 	| Relevance (30) 	| Difficulty Spread (30) 	| Creativity (30) 	| Judge Impression (30) 	| Score 	|
```

Comment:
shouldn't use whitespace to align tables (it doesn't really help anyway when the rows are so long). only use one space between content and the pipes